README.md

```diff
@@ -161,14 +176,19 @@ $ pip install -e pyndustric
 Alternativly, you can run it as module without installation by placing your program file at the root of the project directory.
 
 
-To compile your program, run the [`pyndustric`] module and pass the files to compile as input
-arguments. The compiled program will be printed to standard output. `-` can be used as a file
-to refer to standard input:
+To compile your program, run the [`pyndustric`] module and pass the files to compile as input arguments.
+`-` can be used as a file to refer to standard input:
 
 ```sh
 $ python -m pyndustric yourprogram.py
 ```
 
+The compiled program will be printed to standard output which can be either copeid or redirected to a file:
+
+```
+$ python -m pyndustric yourprogram.py > yourprogram.mlog
+```
```

Comment:
The redirection to files should be obvious to people who have some experience with the shell, so I tried to explain it a bit more for those who actually need this information ("it creates a new file").

Also might as well document the `-c` option. Perhaps the `setup.py` should be updated to support an optional `pip install pyndustric[autocopy]` or something (not in this PR though).

````suggestion
The compiled program will be printed to standard output by default.
You can also redirect the output to a file to create a new file with the compiled program:

```sh
$ python -m pyndustric yourprogram.py > yourprogram.mlog
```

If the optional dependency `autoit` is installed, `-c` or `--clipboard` can be used to
automatically copy the code to the clipboard which allows for very fast edit cycles:

```sh
$ python -m pyndustric -c yourprogram.py
```
````